presto-main/src/main/java/com/facebook/presto/sql/analyzer/StatementAnalyzer.java

```diff
@@ -1535,6 +1538,34 @@ private void analyzeWindowFunctions(QuerySpecification node, List<Expression> ou
 
                 Window window = windowFunction.getWindow().get();
 
+                ImmutableList.Builder<SortItem> orderbyConst = ImmutableList.builder();
+                window.getOrderBy().ifPresent(
+                        orderBy -> orderbyConst.addAll(
+                                orderBy.getSortItems()
+                                .stream()
+                                .filter(item -> item.getSortKey() instanceof Literal).collect(Collectors.toList())));
```

Comment:
We normally use `toImmutableList()` rather than `Collectors.toList()` 
We normally format this as
```
orderBy.getSortItems().stream()
        .filter(...)
        .collect(toImmutableList())
```
You can use 
```
List<SortItem> constSortItems = window.getOrderBy()
        .ifPresent(orderBy -> orderBy.getSortItems()....)
        .orElse(ImmutableList.of());
```

Reply:
Done